pkg/agent/metrics/prometheus.go

```diff
@@ -136,6 +136,16 @@ var (
 		},
 	)
 
+	TotalDenyConnections = metrics.NewGauge(
+		&metrics.GaugeOpts{
+			Namespace:      metricNamespaceAntrea,
+			Subsystem:      metricSubsystemAgent,
+			Name:           "deny_connection_count",
```

Comment:
I think it should be "denied_connection_count" for user-facing terms & messages; it's a "denied connection" not a "deny connection"

Reply:
Makes sense. Updated. Thanks!